src/pages/omOss.tsx

```diff
@@ -0,0 +1,253 @@
+import { useLocale } from "@/locales";
+import { useState } from "react";
+const imagesPath = "/img/groupPictures/";
+
+const prTeam = [
+  [
+    imagesPath + "mathiasPic.jpg",
```

Comment:
How big are the images? If they are too big we might have to minimise them to improve loading speed